test/cli/tarantool_poll.c

```diff
@@ -91,8 +91,10 @@ test_poll() {
 	is(1, ok, "expected non-assert test");
 
 	while (last > -1) {
-		if (streams[last] != NULL)
+		if (streams[last] != NULL) {
 			tnt_close(streams[last]);
+			tnt_stream_free(streams[last]);
+		}
```

Comment:
Please revert all changes unrelated to the fix. They belong to the future PR enabling ASAN.

Reply:
fixed, after this review i create new PR wth this changes and enabling ASAN